.gitignore

```diff
@@ -100,3 +100,6 @@ lib/**/*
 
 # IDE
 .idea
+
+# Transpiled code
+dist/
```

Comment:
Since we are adding this to the `.gitignore`, it would be a good idea to remove the current `dist/` from git (`git rm --cached dist/`).

Reply:
Makes sense! Done in 2db41157cda2749e5cbbd139cf662a40c2f344a1